scripts/lib/api/updateLinks.test.ts

```diff
@@ -127,136 +130,48 @@ describe("updateLinks", () => {
       ]
     `);
   });
-
-  test("update links using a transform function", async () => {
-    const data: HtmlToMdResultWithUrl[] = [
-      {
-        markdown: `
-[link1](algorithms)
-[link2](../apidocs/algorithms)
-[link3](qiskit.algorithms.minimum_eigensolvers.VQE#qiskit.algorithms.minimum_eigensolvers.VQE)
-[link3](../stubs/qiskit.algorithms.minimum_eigensolvers.VQE#qiskit.algorithms.minimum_eigensolvers.VQE)
-[link7](#qiskit_ibm_runtime.RuntimeJob.job)
-          `,
-        meta: {
-          apiType: "class",
-          apiName: "qiskit_ibm_runtime.RuntimeJob",
-        },
-        url: "/docs/api/qiskit-ibm-runtime/stubs/qiskit_ibm_runtime.RuntimeJob",
-        images: [],
-        isReleaseNotes: false,
-      },
-    ];
-
-    const objectsInvEntries = [
-      "stubs/qiskit.algorithms.Eigensolver.html#$",
-      "stubs/qiskit.algorithms.Eigensolver.html#$",
-      "stubs/qiskit.algorithms.EigensolverResult.html#$",
-      "stubs/qiskit.algorithms.EstimationProblem.html#$",
-      "stubs/qiskit.algorithms.EvolutionProblem.html#$",
-      "stubs/qiskit.algorithms.EvolutionProblem.html#$",
-      "stubs/qiskit.algorithms.FasterAmplitudeEstimationResult.html#$",
-      "stubs/qiskit.algorithms.FasterAmplitudeEstimationResult.html#$",
```

Comment:
@frankharkins should I add any of these entries to the single test? I wasn't sure if this is testing something unique.

Reply:
I'm not sure why each line is duplicated, and I don't think it's testing anything unique.

If we remove this, do we have a way of checking `objects.inv` is updated in `updateLinks`?

Reply:
Yeah, the existing test alrealdy has a bunch of stuff for `objects.inv`. Only not `qiskit.algorithms`. I figure you put that there because the custom `transformLink` logic specifically looked at `algorithms`. I think this is all fine.